www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
As far as I know, we have no outstanding bugs for numpy or scipy wheels on Linux or OSX.  We tested the hell out of the numpy and scipy wheels via : https://github.com/matthew-brett/manylinux-testing and http://build.openblas.net .  The wheels are commonly used via travis-ci integration testing.    As far as I know the wheels (using OpenBLAS and Accelerate) are in the same ballpark as MKL packages for speed (not so for Windows of course).   We (the numpy and scipy developers) do support the wheels, but do not directly support the Anaconda / Canopy etc packages.

I think it's reasonable to recommend a big distribution here, as we do in the current text ("For most users ... the easiest way to install"), and it's really the only way to go for Windows, but I think it's too strong to say "especially for OSX".